src/OpenRiaServices.Tools.TextTemplate/Framework/OpenRiaServices.Tools.TextTemplate.csproj

```diff
@@ -135,4 +127,18 @@
   <ItemGroup>
     <Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />
   </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
+    <!-- Required for known type attributes -->
+    <PackageReference Include="System.ServiceModel.Primitives" Version="6.0.0" />
+    <!-- Required for validation attributes -->
+    <PackageReference Include="System.ComponentModel.Annotations" Version="5.0.0" />
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' != 'net6.0' ">
```

Comment:
I suspect we will change net6 and/or add net8 targets within the next year.

Can you rewrite conditions to compare against net4*/net472